.github/workflows/publish.yml

```
@@ -0,0 +1,210 @@
+
+      - name: Update npm
+        run: npm install -g npm@latest
+
+      - run: yarn
+
+      - name: build
+        run: yarn build
+
+      # Prerelease publish steps
+      - name: Bump acala-evm & deploy
+        if: steps.changed-acala-evm.outputs.changed == 'true'
```

actionlint (1.7.9)

property "changed-acala-evm" is not defined in object type {}

(expression)